README.md

```diff
@@ -20,35 +19,85 @@ The frontend portion of the Sheriff Scheduling app (code named Shuber).
-* Run `yarn start` or `npm start` to start up the development server
-   - This should automatically open up a browser to [http://localhost:3000](http://localhost:3000)
+* Install Dependencies
+    - Run `yarn` or `npm install` in the root of the project
+* Start Development Server
+    - Run `yarn start` or `npm start` 
+    - This should automatically open up a browser to [http://localhost:3000](http://localhost:3000)
 
 > See [Available Scripts](#available-scripts) for more info
 
-## Technology Stack Used
-This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
+## Developing Features
+The following process should be used when developing a feature
+* Create a Branch - a branch should be created for any new piece of work
+    - Switch to the branch you would like to branch from (e.g. master)
+    - Create a new branch, named for the feture you are working on using kabob case (_e.g._ my-new-branch)
+* Commit and Sync Often
+    - When working on your branch, commit and sync your changes to github often
+    - Try to logicially group and name your commits; this makes it easier for the PR
+* Create a Pull Request (PR)
+    - Sync your local master with origin
+    - Switch to your working branch
+    - Rebase your working branch (_e.g._ my-new-branch) against master 
+        - Ensure all changes have been comitted and synchronized
+        - Run `git rebase master`
+        - Resolve any conflicts
+        - Run `git rebase --continue`
+        - Continue to resolve the conflicts and continue the rebase, until no conflicts remain (NOTE: the rebase can be ended at any time by running `git rebase --abort`)
+    -  Force Push the rebase onto your remote brach
+        - Run `git push -f`
+    -  Create a Pull Request for your branch from GitHub, add at least one reviewer
+* Make any suggested updates from PR and commit changes
+* Merge branch into master and delete branch
+* From VS Code, prune your branches
+    - Run `git prune`
+* From VS Code, delete your local branch  
+    - Switch to a non-working branch (_e.g._ master) 
+    - Hit `Ctrl+Shift+P`
+    - Start typing `Git: Delete Branch` and select the option when it appears
+    - Select your working branch to delete it
 
-## Project Status
-**todo**
-## Documnentation
+## Conventions
+
+### `Index.ts` Files
+- To reduce the complexity of project structure, this projet does not use `index.ts` files.
 
-**Todo**
-GitHub Pages (https://guides.github.com/features/pages/) are a neat way to document you application/project.
+### Imports
 
-## Security
+When importing more than one item from an external library, place each import on its own line; this makes imports easier to comment out when debugging / testing an idea.
+
+```ts
+import {
+    Glyphicon,
+    Button,
+    Image
+} from 'react-bootstrap'
+```
+vs
+```ts
+import {Glyphicon, Button, Image} from 'react-bootstrap'
+```
+
+### Components
```

Comment:
Maybe a section on containers:

### Containers

Containers are higher level components that wire state from redux into regular Components.  They live in the Containers directory and follow a similar naming convention to Components